### ISSUE TITLE
need genotype explanation on hover

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/monarch-legacy#556]
It would be just fantastic if we could facilitate cross-species understanding by having some kind of informational hover box that told users how to read/understand the genotypes. Each organisms has very unique nomenclature rules and specific issues (for example use of tp53 morpholinos in zebrafish to decrease cell death). These are basically impenetrable to people not in that organism specific field. We could do the model community a nice service by simply explaining these things in a consistent way. 
@bryanlaraway @nlwashington @mbrush

**Comment by nlwashington** [comment]
we could start with a simple FAQ to discuss this.

**Comment by mbrush** [comment]
I totally agree, and had made a ticket a whole back in JIRA about this (https://support.crbs.ucsd.edu/browse/NIF-11759). I think that a good start would be improving the snippets presented in the 'overview' tab for a given model. At present, the overview read as follows for the model/genotype "Vegfa<tm1Pec>/Vegfa<tm1Pec> [involves: 129S1/Sv \* 129X1/SvJ]":

ID: MGI:3047228
GVC: Vegfa<tm1Pec>/Vegfa<tm1Pec>
Extrinsic: undefined
Background: involves: 129S1/Sv \* 129X1/SvJ
Gene(s): Vegfa
Variation(s): <tm1Pec>

It seems simple enough to clarify some of the terms used in the 'decomposition' here (e.g. 'GVC' and 'extrinsic'),  and possibly include hover-over pop-ups that define the genotype components. Happy to work on suggestions here.  

I think a FAQ(or some longer doc) would also be needed to explain the 'common genotype syntax' - its motivation, how to interpret it, and examples of how genotypes in community MOD syntaxes map to it for all of the MODs.
